### ISSUE TITLE
Close and save buttons not closing/saving open tab

### ISSUE THREAD

**Issue by fculpepper** [LD4P/sinopia_editor#2057]
What went wrong? 
When saving, I get confirmation of the save but when closing am told that there is unsaved information. And, when clicking the Close button, the leftmost resource template is closed, not the one that is open.
What did you expect to happen?
I expect the RT that I am working on to save when I press the save button and close when I press the close button.

- What is the impact on your work? Really super annoying.
- What environment are you in? production
- What OS and browser are you using? Windows; Firefox
- If you're cataloging, what Resource Template were you using? UAL Monograph Opus (Un-Nested), UAL Monograph Work (Un-Nested), UAL Monograph Instance (Un-Nested) and UAL Monograph Item (Un-Nested)

**Please include a screenshot**
![Snag_4b45a25](https://user-images.githubusercontent.com/14892870/73020836-abf43400-3de3-11ea-925b-6195b7683d64.png)

![Snag_4b487bd](https://user-images.githubusercontent.com/14892870/73020844-b282ab80-3de3-11ea-9ffa-eabe7265ca5a.png)

![Snag_4b49e72](https://user-images.githubusercontent.com/14892870/73020856-b6163280-3de3-11ea-80cb-1f5b7776d25e.png)

**Comment by peetucket** [comment]
See also #2052 -- this is basically the same thing I believe

**Comment by michelleif** [comment]
#2052 is closed now. @fculpepper are you still seeing this bug?

**Comment by briesenberg07** [comment]
Just [commented](https://github.com/LD4P/sinopia_editor/issues/2052#issuecomment-624303782) on similar issue #2052 .

**Comment by michelleif** [comment]
@fculpepper new version of Sinopia released today that should fix the erroneous message about unsaved info. Can you check if that is fixed? Are you still seeing the problem with closing the wrong tab? (that is also documented in #2183

**Comment by jermnelson** [comment]
Adding a new wrinkle to this bug as it is related to issues with closing/saving the correct tab.

To replicate (when running locally with in `USE_FIXTURES=true`):
1. Open a resource template like `ld4p:RT:bf2:Title:AbbrTitle` and add some content:
![Screen Shot 2020-08-10 at 9 57 17 AM](https://user-images.githubusercontent.com/71847/89803323-e60ef400-daef-11ea-9fd2-1e7f28fdb8bc.png)
2. Open a second resource (do not close the previously opened resource template) like `resourceTemplate:bf2:Note`:
![Screen Shot 2020-08-10 at 9 59 09 AM](https://user-images.githubusercontent.com/71847/89803517-2b332600-daf0-11ea-9f4a-ac0c1c64c051.png)
3. Click on the close button for the first resource template, a pop-up confirmation modal is displayed as expected:
![Screen Shot 2020-08-10 at 10 05 44 AM](https://user-images.githubusercontent.com/71847/89804242-228f1f80-daf1-11ea-9c5b-767d5d24de2c.png)
4. However, if you click on the blue close button, the  
`ld4p:RT:bf2:Title:AbbrTitle` tab is not close but instead the tab with 
the `resourceTemplate:bf2:Note` resource template is closed, leaving the original resource template:

![Screen Shot 2020-08-10 at 10 08 33 AM](https://user-images.githubusercontent.com/71847/89804479-7863c780-daf1-11ea-8751-7c6e30b3b3ee.png)

**Comment by michelleif** [comment]
This seems to be happening again.

I have three templates open. One is a template that I entered some data into "A form about a thing". The other two are resource templates I opened but have not touched, both tabs called "Resource Template".
<img width="1043" alt="Screen Shot 2021-02-18 at 9 29 17 AM" src="https://user-images.githubusercontent.com/23069165/108396826-24d2e480-71cc-11eb-8fbb-6b621a29c248.png">

I close the current highlighted tab and it works as expected. Now I have two tabs.
<img width="1051" alt="Screen Shot 2021-02-18 at 9 29 24 AM" src="https://user-images.githubusercontent.com/23069165/108396862-31efd380-71cc-11eb-9eb1-6da20866e9e5.png">

Now I want to close the "A form about a thing" tab. So I close it and am prompted, as expected, to save the unsaved changes.
<img width="1046" alt="Screen Shot 2021-02-18 at 9 29 31 AM" src="https://user-images.githubusercontent.com/23069165/108396910-40d68600-71cc-11eb-8ed2-f66ae5015e54.png">

I don't want to save, so I just click "Close".

Now I have one template standing, but it's not the one I was on before called "Resource Template". It's the one I tried to close called 
<img width="1044" alt="Screen Shot 2021-02-18 at 9 29 39 AM" src="https://user-images.githubusercontent.com/23069165/108397008-5cda2780-71cc-11eb-97ed-84b7e4e06c3b.png">
"A form about a thing"
